pkg/pillar/dpcreconciler/linuxitems/adapter.go

```diff
@@ -304,6 +313,15 @@ func (c *AdapterConfigurator) Delete(ctx context.Context, item depgraph.Item) er
 		c.Log.Error(err)
 		return err
 	}
+	// Make sure the ethernet interface is DOWN before renaming
```

Comment:
Ditto

Reply:
This method is called when the port is no longer used (by the currently applied DPC). So I think we can just keep it DOWN.